.travis.yml

```diff
@@ -95,6 +91,11 @@ matrix:
         - os: linux
           env: MAIN_CMD='pycodestyle specutils --count' SETUP_CMD=''
 
+        # Try development version of Astropy
```

Comment:
Anything under `allow_failures` has to be included also in the main matrix above. Basically this change has remove the dev testing altogether.
https://github.com/astropy/specutils/pull/418 should address this.